bindings/ruby/ext/ruby_whisper_params.c

```diff
@@ -1303,6 +1325,7 @@ init_ruby_whisper_params(VALUE *mWhisper)
   DEFINE_PARAM(max_len, 11)
   DEFINE_PARAM(split_on_word, 12)
   DEFINE_PARAM(initial_prompt, 13)
+  DEFINE_PARAM(carry_initial_prompt, 36)
```

Comment:
Can you assign `14` to `carry_initial_prompt` and re-number params below?

Reply:
sure thing!

Reply:
Thanks!